.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
Looks like the original `.DS_STORE` is checked in, make sure to remove that too!

Reply:
What do you mean by this? I don't see another .DS_Store in the gitignore.

Reply:
it's https://github.com/MLH-Fellowship/0.5.2-hackathon-geography/blob/0b3eb9711da3db318001ada996ab7c5af70ea65b/.DS_Store